### DISCUSSION TITLE
GENConv

### DISCUSSION THREAD

**Discussion by sincyvt** [pyg-team/pytorch_geometric#8806] [General]
What makes GENConv different from GCNconv

**Comment by rusty1s** [comment]
I don't think they are really comparable (except for the fact that both rely on message passing). In detail, `GENConv`
* uses multi-layer MLPs
* uses custom aggregations
* integrated edge feature support
* uses skip-connections instead of self-loops

**Comment by sincyvt** [comment]
Regarding GENConv, how the model works? any sample figure available

**Reply by rusty1s** [reply]
Did you take a look at the documentation for [`GENConv`](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.GENConv.html#torch_geometric.nn.conv.GENConv). It has the full formula of the operator.
